examples/gatsby/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center">
```

Comment:
We should probably remove the logos from the top since the Nacelle one is out of date and we're not doing this in the other repos, no?

Reply:
Ah, this is just to control the presentation of the docs page on the Gatsby Plugin library. Some other plugins do the same and it looks pretty snappy. Ideally, I'd like to just update our logo, unless you strongly disagree that displaying the logos is a good idea.

Reply:
Ah ok. Let's update the logo on the plugin README, but remove it from the example project if that's cool?

Reply:
Definitely!

Reply:
I'll just remove both, since the new Nacelle logo looks kinda funky next to the Gatsby logo:

<img width="617" alt="Screen Shot 2020-09-10 at 4 51 21 PM" src="https://user-images.githubusercontent.com/5732000/92804298-38a92d80-f386-11ea-80a9-9b4fe75d4dfb.png">